### DISCUSSION TITLE
Any idea about general availability of version 7?

### DISCUSSION THREAD

**Discussion by himanshuz2** [rabbitmq/rabbitmq-dotnet-client#1500] [Q&A] [answered]
Current version is alpha2.  Will appreciate some guidance on expected time frame for general availability of version 7.

**Comment by lukebakken** [comment, marked as answer]
Please watch this milestone. At some point I _might_ add a release date, but there is no schedule at this time:

https://github.com/rabbitmq/rabbitmq-dotnet-client/milestone/53

I'll release a new alpha soon.

@himanshuz2 if you would like to speed up the version 7 release, you could do the following:
* Thoroughly test the version 7 alpha releases.
* Ask if you could contribute code to this client library.
* Purchase a RabbitMQ support contract and explicitly state that version 7 of this client is needed - https://rabbitmq.com/#support
